testing/describes.js

```diff
@@ -429,6 +429,11 @@ class AmpFixture {
     }
     if (spec.extensions) {
       spec.extensions.forEach(extensionId => {
+        if (extensionId.indexOf(':') == -1) {
+            // We default to 0.1 if the user did not give an explicit
+            // version.
+            extensionId = `${extensionId}:0.1`;
```

Comment:
we can't overwrite `extensionId` here  if we can't find an installer. We would need the extension name w/o version number to call `resetScheduledElementForTesting(win, extensionId);`